app/models.py

```diff
@@ -43,6 +43,22 @@ def insert_roles():
     def __repr__(self):
         return '<Role \'%s\'>' % self.name
 
+class IdlingIncident(db.Model):
+    __tablename__ = "incidents"
+    vehicle_id = db.Column(db.Integer, primary_key=True)
+    location = db.Column(db.String(64))
+    date = db.Column(db.DateTime)
+    agency = db.Column(db.String(64))
+    picture = db.Column(db.String(64))
```

Comment:
To answer your question, this should just store a URL. I don't think we want to host photos ourselves.